### PR TITLE
i18n: Romanian translations

### PR DESCRIPTION
**Pull request by BogDAAAMN**
Provided Romanian translations for the keywords available in `components/i18n/GlobalTexts.js` as per #260.

### CODE REVIEW

**Comment by BogDAAAMN** [comment]
@sethburtonhall Maybe delay the merge until we get a second Romanian opinion. cc @ioanaandreeab can you take a look as well please?

File: components/i18n/GlobalTexts.js [hunk 1]
```diff
@@ -190,6 +190,96 @@ const GlobalTexts = {
...
+    PASSWORD: 'Parolă',
+    PASSWORD_FORGOT: 'Ai uitat parola?',
+    PASSWORD_MATCH:
+      'Parola trebuie să conțină cel puțin 8 caractere, o literă mică, o literă mare, o cifră și un caracter special.',
+    PASSWORD_REQUIRED: 'Acesta este un câmp obligatoriu.',
+    PASSWORD_RESET: 'Resetează Parola',
+    PASSWORD_RESET_SUCCESS:
+      'Succes! Verifică-ți emailul pentru a reseta parola.',
+    PLEASE_PROVIDE_FILE_TYPE:
+      "Folosește o imagine .jpg, .jpeg, sau .png pentru logoul companiei. Aceasta va apărea pentru fiecare job disponibil.",
```
Comment (ioanaandreeab) [inline components/i18n/GlobalTexts.js:251]: logo-ul companiei

File: components/i18n/GlobalTexts.js [hunk 2]
```diff
@@ -190,6 +190,96 @@ const GlobalTexts = {
...
+    FILL_OUT: 'Completează-ți profilul ca să poți începe!',
+    FIRST_NAME_REQUIRED: 'Acesta este un câmp obligatoriu.',
+    FROM: 'De la',
+    TO: 'Pentru',
+    GITHUB: 'Autentifică-te cu Github',
+    HERE: 'aici.',
+    HIDE_INFO: 'Ascunde',
+    LAST_NAME: 'Nume',
+    LAST_NAME_REQUIRED: 'Acesta este un câmp obligatoriu.',
+    NAME: 'Nume',
```
Comment (ioanaandreeab) [inline components/i18n/GlobalTexts.js:238]: Prenume

File: components/i18n/GlobalTexts.js [hunk 3]
```diff
@@ -190,6 +190,96 @@ const GlobalTexts = {
         'Vous semblez être nouvelle/nouveau ici! Quel type de compte souhaitez-vous créer?',
     WORK_WITHIN: 'Souhaite travailler dans...',
   },
+  'ro-ro': {
+    ACCOUNT_CREATE: 'Creează un cont nou',
+    ACCOUNT_SIGN_IN: 'Ai deja cont? Autentifică-te',
+    ACCOUNT_SIGN_UP: "Nu ai cont? Inregistrează-te",
```
Comment (ioanaandreeab) [inline components/i18n/GlobalTexts.js:196]: Înregistrează-te

File: components/i18n/GlobalTexts.js [hunk 4]
```diff
@@ -190,6 +190,96 @@ const GlobalTexts = {
...
+    APPLICATIONS_RECEIVED: 'Aplicații Primite',
+    ARCHIVED_LISTINGS: 'Joburi Arhivate',
+    BACK: 'Înapoi la autentificare.',
+    CANDIDATE: 'Candidat',
+    CANDIDATE_SELECT: 'Vreau să mă angajez.',
+    COMPANY: 'Companie',
+    COMPANY_NAME: 'Nume Companie',
+    COMPANY_LOGO: 'Logo Companie',
+    COMPANY_SELECT: 'Vreau să angajez.',
+    COMPANY_WEBSITE: 'Webiste Companie',
```
Comment (ioanaandreeab) [inline components/i18n/GlobalTexts.js:209]: Website

File: components/i18n/GlobalTexts.js (hunk 3 above)
Comment (BogDAAAMN) [reply components/i18n/GlobalTexts.js:196]: oopsie

File: components/i18n/GlobalTexts.js (hunk 4 above)
Comment (BogDAAAMN) [reply components/i18n/GlobalTexts.js:209]: done

File: components/i18n/GlobalTexts.js (hunk 1 above)
Comment (BogDAAAMN) [reply components/i18n/GlobalTexts.js]: arguable dar ok. fixed in 0404d57

**Pull request merged**